.github/actions/unit-tests-action/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Run unit tests
```

Comment:
suggestion (non-blocking): Perhaps we should phrase it differently to reflect that it's not just unit tests but all code quality tools